Assets/MixedRealityToolkit/Inspectors/PropertyDrawers/SceneInfoDrawer.cs

```diff
@@ -1,10 +1,7 @@
-﻿using System;
-using Boo.Lang;
```

Comment:
>Boo.Lang; [](start = 6, length = 9)

What

Reply:
Boo.Lang!?

Reply:
![image](https://user-images.githubusercontent.com/5840182/60357914-bf443b80-9989-11e9-945b-11eac25228fa.png)

Reply:
I think there might be some money in a github app that finds usage of Boo.Lang and posts pictures of Boo